app/controllers/transaction_controller.rb

```diff
@@ -1,15 +1,58 @@
+
 class TransactionController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index
+    @base = Transbank::Onepay::Base::DEFAULT_CALLBACK
+  end
 
   def create
+    Transbank::Onepay::Base.shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
+    Transbank::Onepay::Base.api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
+
+   # ENV["ONEPAY_API_KEY"] = "mUc0GxYGor6X8u-_oB3e-HWJulRG01WoC96-_tUA3Bg"
+    Transbank::Onepay::Base.integration_type = 'TEST'
+    sample_data = JSON.parse '{"items":[{"amount":36000,"quantity": 1,"description":"Fresh Strawberries"},{"amount":16000,"quantity":1,"description":"Lightweight Jacket"}]}'
```

Comment:
Isn't easier/clearer to put this a a hash literal rather than parsing a JSON string?